pytest_mh/utils/fs.py

```diff
@@ -626,7 +626,7 @@ def chmod(self, mode: str, path: str, args: list[str] | None = None) -> SSHProce
         self.backup(path)
         self.logger.info(f'Changing mode bits for "{path}"')
```

Comment:
Since we are here, can you change this to: `Changing mode to {mode} for "{path}"`?